server/data/PrisonLocationResult.ts

```diff
@@ -0,0 +1,24 @@
+interface Agency {
+  agencyId: string
+  description: string
+  agencyType: string
+  active: boolean
+}
+
+interface Location {
+  locationId: number
+  locationType: string
+  description: string
+  locationUsage: string
+  agencyId: string
+  parentLocationId: number
+  currentOccupancy: number
+  locationPrefix: string
+  operationalCapacity: number
+  userDescription: string
+  internalLocationCode: string
+}
+
+export type PrisonLocation = Location
```

Comment:
Oh - is this a TS feature I don't know? Why not just define these as types?

Reply:
Good point, something that came with merging a couple of files but i'll simplify